integration_tests/dbt_project.yml

```diff
@@ -19,7 +19,7 @@ clean-targets: # directories to be removed by `dbt clean`
 
 vars:
   days_back: 30
-  debug_logs: "{{ env_var('DBT_EDR_DEBUG', False) }}"
+  debug_logs: "{{ env_var('DBT_EDR_DEBUG', True) }}"
```

Comment:
Care to explain why did you change the default?
It's just that those logs are pretty spamming and I'm not sure we want them to show in the CI.

Reply:
shouldn't have pushed that change, it was for local work